backend/tsconfig.json

```diff
@@ -6,9 +6,11 @@
         "resolveJsonModule": true,
         "experimentalDecorators": true,
         "allowSyntheticDefaultImports": true,
-        "esModuleInterop": true
+        "esModuleInterop": true,
+        "typeRoots": ["./@types", "node_modules/@types"],
```

Comment:
This may ignore types from the package itself so we will stick to the solution from @b-dabrowski PR